client/additional-methods-setup/upe-preview-methods-selector/enable-upe-preview-task.js

```diff
@@ -46,17 +46,18 @@ const EnableUpePreviewTask = () => {
 					{ interpolateComponents( {
 						mixedString: __(
 							'Get early access to additional payment methods and an improved checkout experience, ' +
-								'coming soon to WooCommerce payments. {{learnMoreLink /}}',
+								'coming soon to WooCommerce payments. {{learnMoreLink /}}.',
 							'woocommerce-payments'
 						),
 						components: {
 							learnMoreLink: (
-								<a href="?TODO">
+								// eslint-disable-next-line max-len
+								<ExternalLink href="https://docs.woocommerce.com/document/payments/additional-payment-methods/#introduction">
```

Comment:
Changed to be an external link component
![Markup 2021-07-21 at 15 58 08](https://user-images.githubusercontent.com/273592/126566171-27c77b49-2c20-43d3-aa59-df1135db7c5f.png)